spec/support/cuprite_setup.rb

```diff
@@ -37,7 +37,9 @@
       window_size: [1200, 800],
       browser_options: remote_chrome ? { 'no-sandbox' => nil } : {},
       inspector: true,
-      headless: !ENV['HEADLESS'].in?(%w[n 0 no false])
+      headless: !ENV['HEADLESS'].in?(%w[n 0 no false]),
+      process_timeout: 20,
+      timeout: 20
```

Comment:
This is an attempt to deal with the `Ferrum::ProcessTimeout` errors that always happen on the first system spec. It seems to help, after adding this, I have not seen the error again, but it would be nice to see if there is some reason the first test takes more than 10 seconds at times.